src/debugger/Debugger.js

```diff
@@ -59,14 +56,14 @@ export class Debugger {
       if (!this.proceedBreakpoint(lineNum, colNum)) return;
 
       console.log("Stopped for breakpoint on line " + lineNum);
-      this.sendDebugInfo(ast, contextStack, lineNum);
+      this.sendDebugInfo(ast, lineNum);
 
       let lastPoll = Date.now();
       let blocking = true;
       let contents = "";
       while (blocking) {
         if (Date.now() - lastPoll > 1000) {
-          contents = fs.readFileSync("./src/debugger/.sessionlogs/proxy2debugger.txt", "utf8").toString().split("\n");
+          contents = this.channel.readIn().toString().split("\n");
           if (contents[0] === "proceed " + lineNum) {
```

Comment:
I'm bemused by the lineNum part of this command. It would be helpful to have single file documenting the protocol.